addons/account/tests/test_chart_template.py

```diff
@@ -363,3 +382,27 @@ def local_get_data(self, template_code):
         self.assertFalse(parent_tax.active, "The parent tax should be inactive.")
         self.assertEqual(len(children_taxes), 2, "Two children should have been created, even if they are inactive.")
         self.assertEqual(children_taxes.mapped('active'), [False] * 2, "Children taxes should be inactive.")
+
+    def test_update_reload_no_new_data(self):
+        """ Tests that the reload does nothing when data are left unchanged.
```

Comment:
As I understand it, you are only checking the ids of the records, not the data itself?

Reply:
Yes, I can add  `assertRecordValues()` but it implies few more loops.